cmritPath.py

```diff
@@ -37,18 +37,19 @@ def __init__(self):
 
         # Hostel Turn node
         G.add_node('hostel_turn', y=12.96696, x=77.71111, street_count=3)
-        
-        #adding edges 
-        G.add_edges_from([('teacher_parking', 'basic_science'),
-                          ('basic_science', 'teacher_parking'),
-                          ('basic_science', 'ganesha_statue'),
-                          ('ganesha_statue', 'basic_science'),
-                          ('ganesha_statue', 'hostel_turn'),
-                          ('hostel_turn', 'ganesha_statue'),
-                          ('volley_ball_court', 'hostel_turn'),
-                          ('hostel_turn', 'volley_ball_court')], length=300)
+
+        # adding edges
+        # G.add_edges_from([('teacher_parking', 'basic_science'),
+        #                   ('basic_science', 'teacher_parking'),
+        #                   ('basic_science', 'ganesha_statue'),
+        #                   ('ganesha_statue', 'basic_science'),
+        #                   ('ganesha_statue', 'hostel_turn'),
+        #                   ('hostel_turn', 'ganesha_statue'),
+        #                   ('volley_ball_court', 'hostel_turn'),
+        #                   ('hostel_turn', 'volley_ball_court')], length=300)
```

Comment:
Commented this.
As we no longer are working by this approach.

Path shall be made btwn two nodes in a stretch only when trees are added.